string.go

```diff
@@ -24,6 +26,36 @@ func Substring[T ~string](str T, offset int, length uint) T {
 	return str[offset : offset+int(length)]
 }
 
+// ChunkString returns an array of strings split into groups the length of size. If array can't be split evenly,
+// the final chunk will be the remaining elements.
+func ChunkString(str string, size int) []string {
```

Comment:
I would use the following prototype instead:

```go
func ChunkString[T ~string](str T, size int) []T
```

Reply:
Makes sense